app/components/related_works/edit_component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module RelatedWorks
+  # Component for editing related works
+  class EditComponent < ApplicationComponent
+    def initialize(form:)
+      @form = form
+      super()
+    end
+
+    attr_reader :form
+
+    def citation_present?
+      form.object.citation.present?
```

Comment:
The logic to determine if the text field seems duplicative of the logic of the js controller. Can you just get rid of it here and call `toggleCitation()` in `connect()` in the js controller?